be/src/olap/tablet.cpp

```diff
@@ -272,16 +255,11 @@ OLAPStatus Tablet::add_rowset(RowsetSharedPtr rowset, bool need_persist) {
     // yiguolei: temp code, should remove the rowset contains by this rowset
     // but it should be removed in multi path version
     for (auto& it : _rs_version_map) {
```

Comment:
it is not correspond with the definition of contains() function

Reply:
Done. 

should be `rowset->version().contains(it.first) `